packages/theme-check-common/src/checks/json-syntax-error/index.ts

```diff
@@ -13,7 +13,7 @@ export const JSONSyntaxError: JSONCheckDefinition = {
     docs: {
       description: 'This check exists to prevent invalid JSON files in themes.',
       recommended: true,
-      url: 'https://shopify.dev/docs/themes/tools/theme-check/checks/valid-json',
+      url: 'https://shopify.dev/docs/storefronts/themes/tools/theme-check/checks/valid-json',
```

Comment:
I get a 404 on this URL, so I believe we may update this one to this:
```suggestion
      url: 'https://shopify.dev/docs/storefronts/themes/tools/theme-check/checks/json-syntax-error',
```